lib/language_pack/helpers/nodebin.rb

```diff
@@ -4,10 +4,11 @@ class LanguagePack::Helpers::Nodebin
   NODE_VERSION = "20.9.0"
   YARN_VERSION = "1.22.19"
 
-  def self.hardcoded_node_lts
+  def self.hardcoded_node_lts(arch: )
+    arch = "x64" if arch == "amd64"
     {
       "number" => NODE_VERSION,
-      "url"    => "https://heroku-nodebin.s3.us-east-1.amazonaws.com/node/release/linux-x64/node-v#{NODE_VERSION}-linux-x64.tar.gz"
+      "url"    => "https://nodejs.org/download/release/v#{NODE_VERSION}/node-v#{NODE_VERSION}-linux-#{arch}.tar.gz",
```

Comment:
(Something for the future) It looks like the `tar.xz` downloads are ~40% smaller than the gzipped tarfiles - I wonder if it would result in a net reduction in build times after accounting for the decompression time difference?
https://nodejs.org/download/release/v22.9.0/